mmv1/provider/file_template.rb

```diff
@@ -80,13 +80,10 @@ def generate(pwd, template, path, provider)
     # template is used to determine metadata about the file based on how it is
     # generated
     def format_output_file(path, template)
-      if path.end_with?('.py') && @env[:pyformat_enabled]
-        run_formatter("python3 -m black --line-length 160 -S #{path}")
-      elsif path.end_with?('.go') && @env[:goformat_enabled]
-        run_formatter("gofmt -w -s #{path}")
+      return unless path.end_with?('.go') && @env[:goformat_enabled]
```

Comment:
This runs `goimports` on every Go file, while prior to this change it was only run on every _generated_ Go file but not handwritten ones (saving import churn and generation time)

Reply:
Oh yup, I just can't read. Sorry!